phantom_token.c

```diff
@@ -811,3 +808,70 @@ static char* set_client_credential_configuration_slot(ngx_conf_t *config_setting
 
     return "invalid_client_credential";
 }
+
+/*
+ * Add the error response as a JSON object that is easier to handle than the default HTML response that NGINX returns
+ * Note that for SPAs, the OAuth Proxy plugin runs before this one and adds CORS headers so that the browser can read this payload
+ * http://nginx.org/en/docs/dev/development_guide.html#http_response_body
+ */
+static ngx_int_t write_error_response(ngx_http_request_t *request, ngx_int_t status, phantom_token_configuration_t *module_location_config)
+{
+    ngx_int_t rc;
+    ngx_str_t code;
+    ngx_str_t message;
+    u_char jsonErrorData[256];
```

Comment:
other variables in this file use `this_format` rather than `thisFormat`. I suggest you rename these variables to be consistent.

Reply:
Done - that was an oversight on my part - checked other vars also